src/strategy/strategies/Othent.ts

```diff
@@ -1,314 +1,126 @@
+import * as othentKMS from '@othent/kms'
+import type { ConnectReturnType } from '@othent/kms'
+import type Transaction from "arweave/node/lib/transaction";
```

Comment:
Let's keep the old import:
```suggestion
import type Transaction from "arweave/web/lib/transaction";
```